### PR TITLE
docs(blog): add this week in databend 96

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: website/blog/2023-06-04-databend-weekly-96.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Virtual Column
+
+One way to optimize querying JSON internal fields is to create virtual columns for these fields using `CREATE VIRTUAL COLCUM`. After creating the virtual columns, we can generate them using `GENERATE VIRTUAL COLUMN` function. Once these steps are completed, we can directly access data in the JSON internal fields through their corresponding virtual columns.
```
Comment (soyeric128) [inline website/blog/2023-06-04-databend-weekly-96.md:41]: One of the ways to optimize querying JSON internal fields is to create and generate virtual columns for these fields with the `CREATE VIRTUAL COLCUM` and `GENERATE VIRTUAL COLUMN` functions. This enables direct queries against the JSON internal fields through their corresponding virtual columns.

File: website/blog/2023-06-04-databend-weekly-96.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- [RFC | JSON Optimization - Virtual Column](https://databend.rs/doc/contributing/rfcs/json-optimization#virtual-column)
+- [PR | feat(query): Support generate virtual columns](https://github.com/datafuselabs/databend/pull/11590)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Interactive Documents Powered by Askbend
+
+Do you remember [AskBend](https://github.com/datafuselabs/askbend)? It is a Knowledge Base Search and Completion project created and maintained by the Databend team, combining the simplicity of Databend with the magic of OpenAI.
```
Comment (soyeric128) [inline website/blog/2023-06-04-databend-weekly-96.md:71]: combining the simplicity of Databend with the magic of OpenAI.   --------Im not sure I understand "simplicity" here

File: website/blog/2023-06-04-databend-weekly-96.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+- Added support for deduplication of insertion/mutation/update based on tags.
+- Added optimization hint support for COPY INTO.
+- Read *[Docs | Deploying a Databend Cluster](https://databend.rs/doc/deploy/metasrv/metasrv-deploy)* to learn about the tips when manually deploying a Databend cluster.
+- Read *[Docs | Data Types - BITMAP](https://databend.rs/doc/sql-reference/data-types/data-type-bitmap)* and *[Docs | Bitmap Functions](https://databend.rs/doc/sql-functions/bitmap-functions/)* to understand the usage of the BITMAP data type in Databend.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Use Loser Tree to Optimize Sorting Algorithm
```
Comment (soyeric128) [inline website/blog/2023-06-04-databend-weekly-96.md:99]: Optimize .. with Loser Tree

File: website/blog/2023-06-04-databend-weekly-96.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+Currently, Databend's sorting algorithm uses MergeSort based on a heap. 
+
+There are three main algorithms for multiple merging: heap sort, winner tree, and loser tree. In all three algorithms, each heap adjustment of the heap sorting requires comparison with the left and right child nodes. The number of comparisons is 2logN while the comparison times for winner tree and loser tree adjustments are logN.
```
Comment (soyeric128) [inline website/blog/2023-06-04-databend-weekly-96.md:103]: In all three algorithms,-> For all of them,

File: website/blog/2023-06-04-databend-weekly-96.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+The difference between them is that the winner tree needs to compare with sibling nodes and update the parent node whereas the loser tree only needs to compare with the parent node resulting in fewer memory access times.
```
Comment (soyeric128) [inline website/blog/2023-06-04-databend-weekly-96.md:105]: compare what?

Comment (PsiACE) [reply website/blog/2023-06-04-databend-weekly-96.md:105]: winner tree and loser tree

Comment (soyeric128) [reply website/blog/2023-06-04-databend-weekly-96.md:105]: The contrast lies in how the two trees operate. In a winner tree, each node must be compared with its sibling nodes and then update the parent node. On the other hand, in a loser tree, a node only requires comparison with the parent node, which leads to fewer instances of memory access.

**Review by soyeric128 — APPROVED** [review]

**Pull request merged**